guides/common/modules/proc_invalidating-jwts-of-other-users.adoc

```diff
@@ -0,0 +1,36 @@
+[id="invalidating-jwts-of-other-users"]
+= Invalidating JWTs of other users
+
+You can invalidate all registration JSON Web Tokens of one or more users.
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-invalidating-jwts-of-other-users[].
+To use the API, see the xref:api-invalidating-jwts-of-other-users[].
+
+.Prerequisites
+* Your {Project} user has a role that grants the `view_users` and `edit_users` permissions.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *Users*.
+. In the row of the user whose registration tokens you want to invalidate, from the actions menu, select *Invalidate JWTs*.
+. In the confirmation window, click *Confirm*.
+
+.Verification
+* The {ProjectWebUI} displays the following message: `Successfully invalidated registration tokens for _the user_.`
+
+[id="cli-invalidating-jwts-of-other-users"]
+.CLI procedure
+* Invalidate all registration tokens of multiple users by running Hammer:
++
+[options="nowrap" subs="+quotes,attributes,verbatim"]
+----
+$ hammer registration-tokens invalidate-multiple --search "_My search query_"
```

Comment:
we also have another command to invalidate token for a single specific user.
it is `hammer registration-tokens invalidate --id "user-id-for-invalidation"